java.lang.ClassNotFoundException crash spike on focus 8.13.0

google play is showing elevated crash rate for 350490945.apk

Stack traces look like this:
```
java.lang.RuntimeException: 
  at android.app.ActivityThread.handleCreateService (ActivityThread.java:4589)
  at android.app.ActivityThread.access$1700 (ActivityThread.java:301)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2137)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:246)
  at android.app.ActivityThread.main (ActivityThread.java:8425)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:596)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1130)
Caused by: java.lang.ClassNotFoundException: 
  at dalvik.system.BaseDexClassLoader.findClass (BaseDexClassLoader.java:207)
  at java.lang.ClassLoader.loadClass (ClassLoader.java:379)
  at java.lang.ClassLoader.loadClass (ClassLoader.java:312)
  at android.app.AppComponentFactory.instantiateService (AppComponentFactory.java:129)
  at androidx.core.app.CoreComponentFactory.instantiateService (CoreComponentFactory.java:1)
  at android.app.ActivityThread.handleCreateService (ActivityThread.java:4568)
```

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -135,11 +135,6 @@
         <service android:name=".session.SessionNotificationService"
             android:exported="false" />
 
-        <service
-            android:name=".utils.ExperimentsSyncService"
-            android:exported="false"
-            android:permission="android.permission.BIND_JOB_SERVICE" />
-
         <meta-data android:name="android.webkit.WebView.MetricsOptOut"
                    android:value="true" />
     </application>

